omr_static_lib/makefile

```diff
@@ -67,13 +67,14 @@ endif
 
 # Prepend the path to the library for relative paths.
 # Absolute paths to object files must be left alone.
+# We clear MAKEFLAGS below because it might contain
+# - '-w' which seems to nullify our use of '--no-print-directory'
+# - job-server context which yields warnings that we don't want
+#   mixed into the 'show-objects' output
```

Comment:
For a formal tone, first person should be avoided in the comments.
```suggestion
# Clear MAKEFLAGS below because it might contain
# - '-w' which seems to nullify the use of '--no-print-directory'; or
# - job-server context which yields warnings that should not
#   be mixed into the 'show-objects' output.
```

Reply:
I'm not sure I agree there's a problem with comments being written in the first person, but hopefully the update comment is acceptable.

Reply:
iirc, many years ago, i had received this feedback from @pshipton. first person is considered informal and not suitable for a professional setting because it weakens credibility.